cv_next/app/cv/[cvId]/components/cvData.tsx

```diff
@@ -7,9 +7,7 @@ import DynamicProfileImage from "@/components/ui/DynamicProfileImage";
 
 export default function CvData({ cv }: { cv: CvModel }) {
   const uploader = JSON.parse(JSON.stringify(cv.user_id));
-  const userName = uploader.username?.length
-    ? uploader.username
-    : uploader.full_name;
+  const userName = uploader.full_name || uploader.username;
```

Comment:
So, again - `full_name` (or display name) doesn't have any API meaning, so hrefing to `/profile/full_name` is meaningless. You should take *only* `uploader.username` - if it doesn't exists then it's an error!

Reply:
Read the code again before you assume the uses of a variable 
I specified the use of username in the href so it is only for display purposes

Reply:
you are right that we need to check uploder.username exist

Reply:
> Read the code again before you assume the uses of a variable I specified the use of username in the href so it is only for display purposes

Correct - then the variable should be easier to distinguish (e.g. `displayName`, not `userName`).